articles/flutter-vs-reactnative/index.md

```diff
@@ -0,0 +1,93 @@
+
+##### A UI first framework
+Flutter has a wide gamut of widgets built right in.  Want to change height, width, color, alignment etc.? Just wrap in an AnimatedContainer and you’re done. Imagine doing all that in any other framework. Flutter widgets are platform agnostic. Hence it's possible to show iOS style translucent dialog boxes within Android in Flutter using the Cupertino package.
+
+#### What React gets right
+##### Oh yes, JS!
+JS is one of the most popular languages among developers. That makes the transition for a web developer into mobile app development smoother.
+
+##### One codebase for mobile, web, and desktop apps
+JS facilitates developers to share code with web apps by creating abstraction components that can compile to target platforms. The world is your oyster in React! For example, Skype developed ReactXp to have a largely single codebase for Android, iOS, web and windows 10 UWP apps.
+
+##### More mature and a community far and wide
+No amount of money and resources can substitute maturity, and React is the most mature framework that is out there. You’ll find it difficult to encounter a question that has not already been answered in Stack Overflow. Anything you need to make your application over the top and rest assured you’ll find a 3rd party library for it.
+
+#### React vs Flutter, fight!
+##### Performance
+Nothing comes close to pure native apps in terms of performance. React Native and Flutter is no exception. But when we compare the two though, Flutter comes on top, because of how it achieves cross-platform. React Native has to use a JS bridge to communicate with native APIs, hence slowing things down. Flutter on the other hand compiles C/C++ library to Dart, since it's closer to metal and it can be consumed faster.
+
+##### Components out of the box
+Many of the common components that you’d need as a developer are not supplied in React Native. You’ll have to hunt down 3rd party libraries to achieve the desired task. Flutter on the other hand, supports a whole ranger of widgets (components). Material design and Cupertino design (Apple’s design philosophy) are supported. There are over 375 official widgets out of the box, and many more being added all the time.
+
+##### App sizes
+One of the bigger tradeoffs with the cross-platform is the final build size of the app. It will always be more than an app built with vanilla Java/Kotlin/Swift. React Native and Flutter apps are no different. But on a comparative basis, React Native apps on average are smaller than Flutter apps. This is not a rule of thumb though.
```

Comment:
*it will always be more than an app? 
Not sure if that would sound better with something around the lines of 
the app will be greater in size? 🤔